source/api_documentation/signal_api/index.rst

```diff
@@ -470,6 +470,80 @@ Note: As signals are applied to the call, the response time of the API will incr
   curl -k -H "Content-Type: application/json" -X POST -d '{"search": {"transaction_id": "00000000-00000001"},"signals": [{"name": "Quality Call","value": "true"}], "custom_data": [{"name": "channel", "value": "Paid Search"}],"oauth_token": <YOUR OAUTH TOKEN>}'  https://invoca.net/api/@@SIGNAL_API_VERSION/transactions.json
   curl -k -H "Content-Type: application/json" -X POST -d '{"search": {"transaction_id": "00000000-00000001"},"signals": [{"name": "Appointment Made","partner_unique_id": "1","occurred_at_time": "1440607313","value": "false"}], "custom_data": [{"name": "channel", "value": "Paid Search"}],"oauth_token": <YOUR OAUTH TOKEN>}'  https://invoca.net/api/@@SIGNAL_API_VERSION/transactions.json
 
+New Signal Transactions
```

Comment:
A few high level comments:

I wonder if we should leave with New Signal Transactions? Or maybe we do after we have more customers live... 

Either way, there should be a corresponding section on `Updates and Idempotency` like there is above, where this mentions how it's only based on partner_unique_id. Please include a note on revenue is added when new PUID is used, versus updated on existing PUID if using the same PUID. 

Best practices above don't apply the same as down here (so maybe up on L461 indicate that is only if not on the new system?)

Reply:
Re ordered it so the order is
... (previous docs) ...
* Signal Transactions Rollup
* Updates and Idemptotency
* Best Practices
...

Added change notices to those two sections regarding Signal Transactions Rollup